charts/n8n/Chart.yaml

```
@@ -34,5 +34,7 @@
   artifacthub.io/prerelease: "false"
   # supported kinds are added, changed, deprecated, removed, fixed and security.
   artifacthub.io/changes: |
-    #- kind: changed
-    #  description: "fix .."
+    - kind: added
+      description: "Incorporate features and bug fixes from n8n 1.86.0"
+    - kind: added
+      description: "Add additional example helm values for how to add n8n main configuration settings to the values"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)